src/data/mutable_data.rs

```diff
@@ -308,3 +318,252 @@ impl Debug for MutableData {
                self.owners)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    extern crate rand;
```

Comment:
I think this crate import should be covered in lib.rs.  Just a `use` statement should work here.